lib/fluent/event.rb

```diff
@@ -70,6 +88,10 @@ def repeatable?
       true
     end
 
+    def slice(index, num)
+      self.dup
```

Comment:
I can't understand what you mean.
But, I think that `OneEventStream#slice` should return empty stream for first argument `>=1`.